internal/config/from_env_test.go

```diff
@@ -263,7 +264,7 @@ func TestInheritEnv(t *testing.T) {
 			GITHUB_TOKEN: "ENVTOKEN",
 			hostname:     "github.com",
 			wants: wants{
-				hosts:     []string{"github.com", "example.org"},
+				hosts:     []string{"example.org", "github.com"},
```

Comment:
Using `set` causes the ordering to change, not sure if that makes a difference though